orchestrator/cli/database.py

```diff
@@ -194,3 +196,49 @@ def history(
 
     """
     command.history(alembic_cfg(), verbose=verbose, indicate_current=indicate_current)
+
+
+@app.command(help="Create revision based on diff_product_in_database.")
+def migrate_domain_models(
+    message: Optional[str] = typer.Option("update domain models", help="Migration name"),
+    test: Optional[bool] = typer.Option(False, help="Optional boolean if you don't want to generate a migration file"),
+    inputs: Optional[str] = typer.Option("{}", help="stringified dict to prefill inputs"),
+) -> Union[Tuple[List[str], List[str]], None]:
+    """Create migration file based on SubscriptionModel.diff_product_in_database. BACKUP DATABASE BEFORE USING THE MIGRATION!.
+
+    You will be prompted with inputs for new models and resource type updates.
+
+    with the above feature, inputs is mostly for testing to prefill the given inputs, but here examples:
```

Comment:
The inputs argument is mostly used for testing the prefill of given inputs, here examples:
(denk ik)

Reply:
reads better yea, will update.